### ISSUE TITLE
Create a Slack Group and Link it to Index Page

### ISSUE THREAD

**Issue by iarobinson** [iarobinson/learn-to-be-a-useful-programmer#23]
Since we're all making progress with this, I think it makes sense to have the ability to chat on the side. If one of you would like to set up a slack group (maybe call it `learn-to-be-a-useful-programmer.slack.com`).

- [x] Set up the slack channel via slack.com
- [x] Add a button to the top of the index.html page which opens the slack channel in a new window
- [x] Invite Me!

If you want to take on this project, please comment below first. We don't want two people doing this at the same time.

**Comment by makromikro** [comment]
@iarobinson I agree. This was actually what I was looking for. I don't have twitter to communicate with you faster and the communication through github is a little slow. A chat integrated to it would be much better. I can do this!

**Comment by makromikro** [comment]
@iarobinson I created the slack channel. Here is the invite link: https://join.slack.com/t/useful-programmer/shared_invite/enQtNTUyNDMwNTUyNzA3LTQ5OGFiMjEyNjIwOWRlYjYwYmE2ZmQ5OTA2ZmIwMjI0M2QxNDI2OGJjYzcyZjM2NWU0ZTRiNTRmMTQwNzc3Nzc (I didn't have your email address).

I couldn't name the channel learn-to-be-a-useful-programmer.slack.com because it is longer than 21 letters and they wont accept it, instead it is useful-programmer.slack.com. I also added the link to the index.html but not to the very top. I hope this looks good.
